test/pipelines-it-remote.yml

```diff
@@ -0,0 +1,59 @@
+jobs:
+- job: 'integration_test_remote'
+  pool: 'NNI CI KUBE CLI'
```

Comment:
Remoe pool in our yml file. Suggest to configure it in Specific Azure pipeline.

Reply:
removed in yml file.